incubator/java-openliberty/image/project/Dockerfile

```diff
@@ -64,6 +64,9 @@ FROM openliberty/open-liberty:{{.stack.libertyversion}}-kernel-java8-openj9-ubi
 #2a) copy any resources 
 COPY --chown=1001:0 --from=compile /shared /opt/ol/wlp/usr/shared/
 
+# Work around https://github.com/moby/moby/issues/37965
```

Comment:
```suggestion
# Work around https://github.com/moby/moby/issues/37965.  It seems the problem may be triggered when the previous COPY command adds no contents to the next layer (because it was empty). 
```

Reply:
@matthewpwilson seems there's some more specific guidance to share here.... which is the fact this is triggered because the previous COPY added nothing.  Not sure I like my wording.. what do you think?